source/loader/layers/sanitizer/linux/sanitizer_utils.cpp

```diff
@@ -97,4 +69,17 @@ std::string DemangleName(const std::string &name) {
     return result;
 }
 
+std::string RunCommand(const char *cmd) {
```

Comment:
Do we absolutely need to run a binary this way? There's no way around it? How is `cmd` sanitized?